extras/basic_example/public/script.js

```diff
@@ -93,7 +94,7 @@ window.onload = function () {
   createToken(roomData, function (response) {
     var token = response;
     console.log(token);
-    room = Erizo.Room({token: token});
+    room = Erizo.Room({token: token, forceCodecs: forceCodecs});
```

Comment:
Why do we need another flag for this? why not using the current mediaConfiguration parameter?

Reply:
the idea is that the configuration is retrieved inside the token, but you choose if you want to force the codecs. this is meant specifically for the p2p sessions. There are case where I specify forceCodecs and so all my clients uses what i specified in the mediaconfiguration. There are other situations in which I don't want to force codecs, so in a p2p many 2 many for example, each client signal what it's best for him.

Reply:
The idea of the mediaConfiguration is to force codecs in a room. If you want to do it by client side I think we would need different API and I wouldn't use the same mediaConfiguration feature, which is tied to the Room. Anyway, I don't know in which scenario the mediaConfiguration wouldn't work (without forceCodecs). You can always add a set of "allowed" codecs to the mediaConfiguration object, and the selected codec will be part of the result of overlapping mediaConfiguration with the codecs included in the both sender and receiver SDPs.